admin_manual/configuration/server/caching_configuration.rst

```diff
@@ -1,267 +1,402 @@
+--------------------------
 
-+---------------------------------------------+------------------------------------------------------------------+
-| Setup/Configuration                         | Result                                                           |
-+=============================================+==================================================================+
-| If file locking is enabled, but the locking | The application will not be usable                               |
-| cache class is missing, then an exception   |                                                                  |
-| will appear in the web UI                   |                                                                  |
-+---------------------------------------------+------------------------------------------------------------------+
-| If file locking is enabled and the locking  | There will be a white page/exception in web UI. It               |
-| cache is configured, but the PHP module     | will be a full page issue, and the application will not be       |
-| missing.                                    | usable                                                           |
-+---------------------------------------------+------------------------------------------------------------------+
-| All enabled, but the Redis server is not    | The application will be usable. But any file operation will      |
-| running                                     | return a **"500 Redis went away"** exception                     |
-+---------------------------------------------+------------------------------------------------------------------+
-| If Memcache is configured for "local" and   | There will be a white page and an exception written to the logs, |
-| "distributed", but the class is missing     | This is because autoloading needs the missing class. So there is |
-|                                             | no way to show a page                                            |
-+---------------------------------------------+------------------------------------------------------------------+
+The caching backends supported by ownCloud are:
+
+* :ref:`APCu <apcu-label>`: This is a local cache for systems running PHP 5.6 and up. APCu 4.0.6 and up is required. Alternatively you can use `the Zend OPCache`. However, **it is not a data cache**, only an opcode cache.
+* :ref:`Redis <redis-label>`: This is a distributed cache for multi-server ownCloud installations. Version 2.2.6 or higher of the PHP Redis extension is required.
+* :ref:`Memcached <memcached-label>`: This is a distributed cache for multi-server ownCloud installations.
+
+.. note:: 
+   You may use *both* a local and a distributed cache. 
+   The recommended ownCloud caches are APCu and Redis. 
+
+.. note::
+   If you do not install and enable a local memory cache you will see a warning on your ownCloud admin page. A memory cache is not required and you may safely ignore the warning if you prefer.
```

Comment:
I believe that was in the original text (well, a slightly different version of it). Thanks for the tip. I'll take it out.